worlds/yugioh06/Options.py

```diff
@@ -0,0 +1,179 @@
+    option_September_2003 = 1
+    option_March_2004 = 2
+    option_September_2004 = 3
+    option_March_2005 = 4
+    option_September_2005 = 5
+    default = option_September_2005
+
+
+class FinalCampaignBossUnlockCondition(Choice):
+    """How to unlock the final campaign boss (Who is the goal)"""
+    display_name = "Final Campaign Boss unlock Condition"
+    option_campaign_opponents = 0
+    option_challenges = 1
+
+
+class FourthTier5UnlockCondition(Choice):
+    """How to unlock the forth campaign boss"""
+    display_name = "Fourth Tier 5 Campaign Boss unlock Condition"
+    option_campaign_opponents = 0
+    option_challenges = 1
+
+
+class ThirdTier5UnlockCondition(Choice):
+    """How to unlock the third campaign boss (Which is the goal)"""
+    display_name = "Third Tier 5 Campaign Boss unlock Condition"
+    option_campaign_opponents = 0
+    option_challenges = 1
+
+
+class FinalCampaignBossChallenges(Range):
+    """Number of Limited/Theme Duels completed for the Final Campaign Boss to appear"""
+    display_name = "Final Campaign Boss challenges unlock amount"
+    range_start = 0
+    range_end = 91
+    default = 10
+
+
+class FourthTier5CampaignBossChallenges(Range):
+    """Number of Limited/Theme Duels completed for the Forth Level 5 Campaign Opponent to appear"""
+    display_name = "Fourth Tier 5 Campaign Boss unlock amount"
+    range_start = 0
+    range_end = 91
+    default = 5
+
+
+class ThirdTier5CampaignBossChallenges(Range):
+    """Number of Limited/Theme Duels completed for the Third Level 5 Campaign Opponent to appear"""
+    display_name = "Third Tier 5 Campaign Boss unlock amount"
+    range_start = 0
+    range_end = 91
+    default = 2
+
+
+class FinalCampaignBossCampaignOpponents(Range):
+    """Number of Campaign Opponents Duels defeated for the Final Campaign Boss to appear"""
+    display_name = "Final Campaign Boss campaign opponent unlock amount"
+    range_start = 0
+    range_end = 24
+    default = 12
+
+
+class FourthTier5CampaignBossCampaignOpponents(Range):
+    """Number of Campaign Opponents Duels defeated for the Forth Level 5 Campaign Opponent to appear"""
+    display_name = "Fourth Tier 5 Campaign Boss campaign opponent  unlock amount"
+    range_start = 0
+    range_end = 23
+    default = 7
+
+
+class ThirdTier5CampaignBossCampaignOpponents(Range):
+    """Number of Campaign Opponents Duels defeated for the Third Level 5 Campaign Opponent to appear"""
+    display_name = "Third Tier 5 Campaign Boss campaign opponent unlock amount"
+    range_start = 0
+    range_end = 22
+    default = 3
+
+
+class NumberOfChallenges(Range):
+    """Number of random Limited/Theme Duels that are included. The rest will be unaccessible."""
+    display_name = "Number of Challenges"
+    range_start = 0
+    range_end = 91
+    default = 10
+
+
+class StartingMoney(Range):
+    """The amount of money you start with"""
+    display_name = "Starting Money"
+    range_start = 0
+    range_end = 99999999
```

Comment:
may want to pick a more reasonable range_end as this makes the UI on player options incredibly hard to manipulate

Reply:
changed it to 100.000